tests/dtypes_test.py

```diff
@@ -578,6 +566,34 @@ def test_check_dtype_array(self):
     with self.assertWarnsRegex(DeprecationWarning, msg):
       jax.jit(dtypes.check_user_dtype_supported)(x)
 
+  @parameterized.parameters([True])  # TODO(mattjj): make jit=False work
+  def test_primal_tangent_dtype(self, jit):
+    dt = dtypes.primal_tangent_dtype(jnp.int8, jnp.bfloat16)
```

Comment:
Why isn't `dt` ever used?

Reply:
It is, on the first line of the function `h`!